plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1290,12 +1290,12 @@ preferences.section.bedrock.description=On this page you can tweak the setting r
 preferences.section.notifications=Notifications
 preferences.section.notifications.description=On this page you can change settings regarding notifications in MCreator.
 preferences.ui.language=Interface language
+preferences.ui.language.description=WARNING: Translations are community-contributed and may be missing, inaccurate or wrong.<br>\
+  You need to restart MCreator to apply changes.
 preferences.ui.usemacOSMenuBar=Use the default macOS menu bar
 preferences.ui.usemacOSMenuBar.description=On a macOS device, use the default menu bar located on the top of your screen.
 preferences.ui.useNativeFileChooser=Use default system file chooser
 preferences.ui.useNativeFileChooser.description=Check to use the default system file chooser. Only works on Windows at the moment.
-preferences.ui.language.description=WARNING: This is a BETA feature. Many translations are still missing. Translations are community<br>\
```

Comment:
I think you should keep `Many translations are still missing.` as not every language translated everything. Also, most of languages do not translate tool tips.

Reply:
Also part about community

Reply:
The community part is on line 1293.